packages/terra-dropdown-button/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ ChangeLog
 
 Unreleased
 ----------
+### Fixed
+* Fixes Primary click button not closing menu, when menu is open
+* Fixes multiple instances of split-button and drop-down opening simultaneously.
+
+###Added
```

Comment:
This needs a space between the hashes and text to format correctly as markdown.

Reply:
Updated here ca75d9ad660c420f613fc22fa90a6b70ce0f8763